delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/model/ArchiveRejection.scala

```diff
@@ -131,6 +135,22 @@ object ArchiveRejection {
     */
   final case class WrappedFileRejection(rejection: FileRejection) extends ArchiveRejection(rejection.reason)
 
+  /**
+    * Rejection returned when attempting to evaluate a command but the evaluation took more than the configured maximum value
+    */
+  final case class ArchiveEvaluationTimeout(command: CreateArchive, timeoutAfter: FiniteDuration)
```

Comment:
We could have only one rejection covering both which would take the EvaluationError.
From what I see, we don't process them differently and if we want to the EvaluationError can help us for that.
We may also want EvaluationError not to extend Exception so as not to be able to hide them

Reply:
I made it this way because It gives a better message to the client. If I have to implement the message in the common rejection is going to be as verbose:
```scala
final case class ArchiveEvaluationTimeout(err: EvaluationError) extends ArchiveRejection(err match {
case ...
})
```

So I don't see any benefit from it.

About the `Exception`, yes I will remove it